mmseg/models/losses/dice_loss.py

```diff
@@ -69,28 +69,38 @@ class DiceLoss(nn.Module):
         ignore_index (int | None): The label index to be ignored. Default: 255.
     """
 
-    def __init__(self,
-                 loss_type='multi_class',
-                 smooth=1,
-                 exponent=2,
-                 reduction='mean',
-                 class_weight=None,
-                 loss_weight=1.0,
-                 ignore_index=255):
+    def __init__(
+            self,
+            #  loss_type='multi_class',
+            smooth=1,
+            exponent=2,
+            reduction='mean',
+            class_weight=None,
+            loss_weight=1.0,
+            ignore_index=255,
+            **kwards):
         super(DiceLoss, self).__init__()
-        assert loss_type in ['multi_class', 'binary']
-        if loss_type == 'multi_class':
-            self.cls_criterion = dice_loss
-        else:
-            self.cls_criterion = binary_dice_loss
+        # assert loss_type in ['multi_class', 'binary']
+        # if loss_type == 'multi_class':
+        #     # self.cls_criterion = dice_loss
+        #     self.binary_cls = False
+        # else:
+        #     # self.cls_criterion = binary_dice_loss
+        #     self.binary_cls = True
+        # self.cls_criterion = dice_loss
```

Comment:
Why comment these lines?

Reply:
Cause `dice_loss()` method has handled these two cases, I think it's no need to add extra codes.

Reply:
In this case, we could delete this code.